### PR TITLE
added a whole bunch of quotes

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by wpoely86** [comment]
this should be fine

File: base.sh [hunk 1]
```diff
@@ -362,18 +360,18 @@ firststart () {
...
 
     echo 0 > $chkid
-    if [ -f $chkprestage ]
+    if [ -f "$chkprestage" ]
     then
-	   $chkprestage
+	   "$chkprestage"
     fi
-    cr_run -- ./$scriptname > ${jobout} 2> ${joberr} &
+    cr_run -- "./$scriptname" > ${jobout} 2> ${joberr} &
```
Comment (boegel) [inline base.sh]: why not around `${jobout}`?

File: base.sh [hunk 2]
```diff
@@ -407,18 +405,18 @@ makechkpt () {
...
     then
-    	chkfile_curTime=`stat -c %%Y $chkfile`
+        # double percent, based this script is pushed through Python
+    	chkfile_curTime=`stat -c %%Y "$chkfile"`
     else
     	chkfile_curTime=0
     fi
     myecho "chkfile_lastTime: $chkfile_lastTime; chkfile_curTime: $chkfile_curTime"
-    if [ -f $chkfile ] && [ $chkfile_curTime -gt $chkfile_lastTime ]
+    if [ -f "$chkfile" ] && [ $chkfile_curTime -gt $chkfile_lastTime ]
```
Comment (boegel) [inline base.sh]: also quotes around `$chkfile_lastTime` and `$chkfile_curTime`?

File: base.sh [hunk 3]
```diff
@@ -446,19 +444,19 @@ endofjob () {
     myecho
     myecho "begin $fun `date`"
     # Add stdout and stderr to central files
-    if [ -f $chkpoststage ]
+    if [ -f "$chkpoststage" ]
     then
-       cp -a $chkpoststage /tmp/poststage
+       cp -a "$chkpoststage" /tmp/poststage
```
Comment (boegel) [inline base.sh]: at least use `/tmp/$USER`? (maybe in a next PR)

**Comment by boegel** [comment]
Looks OK, just a couple of places where no quotes were added, unclear why?

File: base.sh (hunk 3 above)
Comment (wpoely86) [inline base.sh]: I will tackle this in my changes.

Comment (boegel) [inline base.sh]: Maybe do the same with the quote remarks, and just merge this one? I'm fine with that.

**Comment by JensTimmerman** [comment]
good catch, no particular reason I think, @wpoely86 feel free to merge this in if you're planning on addressing the remarks.

**Comment by wpoely86** [comment]
okay, I will fix them. Thanks @JensTimmerman

**Pull request merged**